src/pyudev/device/_device.py

```diff
@@ -370,7 +370,8 @@ def from_path(cls, context, path):  # pragma: no cover
         import warnings
 
         warnings.warn(
-            "Will be removed in 1.0. Use equivalent Devices method instead.",
+            "The from_path method is deprecated and "
+            "will be removed in 1.0. Use equivalent Devices method instead.",
```

Comment:
```suggestion
            "will be removed in 1.0. Use Devices.from_path method instead (Device with an s).",
```

might be even better.... i'm happy to propagate this idea throughout.